packages/plugin-essentials/sources/commands/explain/peerRequirements.ts

```diff
@@ -54,43 +54,40 @@ export default class ExplainPeerRequirementsCommand extends BaseCommand {
       });
     }
 
-    const report = await StreamReport.start({
-      configuration,
-      stdout: this.context.stdout,
-      includeFooter: false,
-    }, async report => {
-      const sortCriterias: Array<(opts: [string, PeerRequirement]) => string> = [
-        ([, requirement]) => structUtils.stringifyLocator(project.storedPackages.get(requirement.subject)!),
-        ([, requirement]) => structUtils.stringifyIdent(requirement.requested),
-      ];
+    const sortCriteria: Array<(opts: [string, PeerRequirement]) => string> = [
```

Comment:
Nit:
- `criterion` is the singular
- `criteria` is the plural but can also be used as an informal singular
- `criterias` doesn't exist

Reference: https://www.merriam-webster.com/dictionary/criteria#usage-1